lib/templates/error.html

```diff
@@ -1085,14 +1085,16 @@
     </header>
     <main class="container">
       <article class="content">
+        <div onclick="fetch('/ember_open_editor_for_template_compilation_error')">
         <h1 class="error-header">Dang! Looks like a {{errorType}}.</h1>
-        {{#if location.file}}
-          <h2 class="error-message">{{location.file}}{{#if location.line}}:{{location.line}}{{/if}}</h2>
-        {{/if}}
+          {{#if location.file}}
+            <h2 class="error-message">{{location.file}}{{#if location.line}}:{{location.line}}{{/if}}</h2>
+          {{/if}}
```

Comment:
I think the `onclick` behavior should only exist on the displayed filepath, not on the codeframe and the title

Reply:
I believe users might click on both the file name and the message(`unclosed element...`) in this screenshot.

<img width="395" alt="screen shot 2018-07-11 at 8 00 36 am" src="https://user-images.githubusercontent.com/7725225/42547931-20713c5e-84e2-11e8-8f18-ea50e9e79554.png">

Reply:
I don't think clicking on the message should open the file. That would be very surprising to me and would also prevent e.g. copy-pasting the error message.